info-compiler/src/main/java/com/google/sps/data/State.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** A class of state abbreviation and state name pairs. */
+public class State {
+  public static final Map<String, String> abbreviationToName = new HashMap<>(50);
+  static {
+    abbreviationToName.put("AL", "Alabama");
```

Comment:
Include a sanity test that you have 50 entries in this map :)

Reply:
Will do!